app/views/shelters/edit.html.erb

```diff
@@ -7,5 +7,5 @@
   <%= text_field_tag :state, "#{@shelter.state}" %>
   <%= text_field_tag :zip, "#{@shelter.zip}" %>
 
+  <%= submit_tag "Submit"%>
```

Comment:
Button_to changed to submit_tag and was moved inside form block. Sumbit tag redirects all info in text field tags to whatever path is declared in form tag.